packages/components/button/stories/button.stories.js

```diff
@@ -1,23 +1,80 @@
 import { storiesOf, html } from '@open-wc/demoing-storybook';
-import { withKnobs, text, select } from '@storybook/addon-knobs';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
 import '@tradeshift/elements';
+import { helpers } from '@tradeshift/elements';
 import '@tradeshift/elements.button';
 
+import { types, sizes } from '../src/utils';
+import icons from '../../icon/src/assets/icons';
+
 storiesOf('ts-button', module)
 	.addDecorator(withKnobs)
-	.add('type', () => {
+	.add('default', () => {
+		const type = select(
+			'Type',
+			{
+				default: '',
+				...helpers.objectKeysChangeCase(types)
+			},
+			''
+		);
+		const size = select(
+			'Size',
+			{
+				default: '',
+				...helpers.objectKeysChangeCase(sizes)
+			}
+			, ''
+		);
+
+		const label = text('Lable', 'Button');
+		const busy = boolean('Busy', false);
+		const disabled = boolean('Disabled', true);
+
+		return html`
+			<ts-button 
+				?disabled="${disabled}" 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ATrailingWhitespace)